src/WidgetMetadataEditor.jsx

```diff
@@ -10,7 +10,7 @@ if (metadata === null) {
 return (
   <Widget
     key={widgetPath}
-    src="${REPL_MOB}/widget/MetadataEditor"
+    src="${REPL_ACCOUNT}/widget/MetadataEditor"
```

Comment:
@marcinbodnar is this was just a typo?

Reply:
What do you mean?

Reply:
`near/widget/MetadataEditor` is the proper metadata editor widget - `mob.near/widget/MetadataEditor` is not working properly.

Reply:
Got it, thank you!